kymatio/scattering1d/backend/tensorflow_backend.py

```diff
@@ -113,16 +89,15 @@ def unpad(x, i0, i1):
 
 
 
-backend = namedtuple('backend', ['name', 'modulus_complex', 'subsample_fourier', 'real', 'unpad', 'fft', 'concatenate'])
+backend = namedtuple('backend', ['name', 'modulus', 'subsample_fourier', 'unpad', 'fft', 'concatenate'])
 backend.name = 'tensorflow'
-backend.modulus_complex = Modulus()
+backend.modulus = Modulus()
 backend.subsample_fourier = subsample_fourier
-backend.real = real
 backend.unpad = unpad
 backend.cdgmm = cdgmm
 backend.pad = pad
-backend.pad_1d = pad_1d
-backend.fft = FFT(lambda x: tf.signal.fft(x, name='ifft1d'),
+backend.fft = FFT(lambda x: tf.signal.fft(x, name='fft1d'),
+                  lambda x: tf.signal.fft(tf.cast(x, tf.complex64), name='rfft1d'),
```

Comment:
So there's no speedup in TF for real FFTs?

Reply:
There is a rfft function, however we don't get the correct size from the output of the tf.rfft function.

Reply:
What size does it give?

Reply:
Doesn't appear to be, the size output by the rfft in tf is 513.

Reply:
As opposed to? 1024? So it's a one-sided transform?

Reply:
Yeah. is it possible to attach/repeat something in order to get the full size?